mobile/ios/ios-proxy-services.ts

```diff
@@ -392,13 +391,17 @@ export class IOSSyslog {
 	}
 
 	public read(): void {
+		var shouldLog = false;
+		setTimeout(() => shouldLog = true, 2500);
+
 		var printData = (data: NodeBuffer) => {
-			var output = ref.readCString(data, 0);
-			if(this.matchRegex.test(output)) {
-				this.$logger.out(output);
+			if(shouldLog) {
+				var output = ref.readCString(data, 0);
+				process.stdout.write(output);
```

Comment:
You can use `$logger.write`
https://github.com/telerik/mobile-cli-lib/blob/f1a6b5d4c1c4b3d632d0f95a1b4c053e1768eaa4/logger.ts#L78